packages/@o3r/design/cli/build-design-token.cli.cts

```diff
@@ -48,6 +48,7 @@ void (async () => {
   )).reduce<DesignTokenVariableSet>((acc, { file, parsed }) => {
     parsed.forEach((variable, key) => {
       if (acc.has(key)) {
+        // eslint-disable-next-line no-console -- no logger available
```

Comment:
should not be disable for all files matching `*.cli.cts` or `cli/*.cts` ?

Reply:
depends sometimes we have a winstom logger sometimes not...